lua/neo-tree/utils/init.lua

```diff
@@ -1018,11 +1018,19 @@ end
 M.escape_path_for_cmd = function(path)
   local escaped_path = vim.fn.fnameescape(path)
   if M.is_windows then
-    -- on windows, any punctuation preceeded by a `\` needs to have a second `\`
-    -- added to preserve the path separator. this is a naive replacement and
+    -- on windows, some punctuation preceeded by a `\` needs to have a second
+    -- `\` added to preserve the path separator. this is a naive replacement and
     -- definitely not bullet proof. if we start finding issues with opening files
-    -- or changing directories, look here first.
-    escaped_path = escaped_path:gsub("\\%p", "\\%1")
+    -- or changing directories, look here first. #1382 was the first regression
+    -- from the implementation that used lua's %p to match punctuation, which
+    -- did not quite work. the following characters were tested on windows to
+    -- be known to require an extra escape character.
+    for _, c in ipairs({ "&", "(", ")", ";", "^", "`" }) do
```

Comment:
Alternatively, we could just use a table with pre-escaped patterns like:

```lua
for _, c in ipairs({ "&", "%(", "%)", ";", "%^", "`" }) do
```

And then do the substitution with:

```lua
escaped_path = escaped_path:gsub("\\" .. c, "\\%1")
```

I'm just not intimately familiar with lua patterns (e.g., [magic characters](https://www.lua.org/pil/20.2.html)) so I used a what I thought might be the "safest" approach. If someone puts a `p` (or any other character class like `a`, `d`, etc.) in this table we'll have problems though... :)

Reply:
I think it reads clearer the way you have it now.